org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/jobs/AnalyzeProjectJob.java

```diff
@@ -453,88 +465,21 @@ public void uncaughtException(Thread th, Throwable ex) {
     return wrapped;
   }
 
-  private static Integer computeChecksum(IDocument iDoc, Issue issue) throws BadLocationException {
+  private static Integer computeChecksum(IDocument iDoc, Issue issue) {
     Integer checksum;
     Integer startLine = issue.getStartLine();
     if (startLine == null) {
       checksum = null;
     } else {
-      Range rangeInFile = SonarMarker.findRangeInFile(issue, iDoc);
-      checksum = SonarMarker.checksum(rangeInFile.getContent());
+      Range rangeInFile;
+      try {
+        rangeInFile = SonarMarker.findRangeInFile(issue, iDoc);
+        checksum = SonarMarker.checksum(rangeInFile.getContent());
+      } catch (BadLocationException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1166)